Source/Core/DolphinWX/Config/AudioConfigPane.cpp

```diff
@@ -241,3 +262,11 @@ void AudioConfigPane::PopulateBackendChoiceBox()
   int num = m_audio_backend_choice->FindString(StrToWxStr(SConfig::GetInstance().sBackend));
   m_audio_backend_choice->SetSelection(num);
 }
+
+void AudioConfigPane::AnyBackendWithLatencyControl()
```

Comment:
This probably shouldn't be named in question-form if it doesn't actually return a result.